docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -265,7 +279,23 @@ When configuring the Kotlin daemon's JVM arguments, note that:
   > even if other requested JVM arguments are different, this daemon will be reused instead of starting a new one.
   >
   {type="note"}
-* If the `Xmx` argument is not specified, the Kotlin daemon will inherit it from the Gradle daemon.
+
+If the following arguments are not specified, the Kotlin daemon will inherit them from the Gradle daemon:
+
+* -Xmx
+* -XX:MaxMetaspaceSize
+* -XX:ReservedCodeCacheSize (Defaults to 320m if not inherited and unspecified)
+
+The Kotlin daemon also has a number of defaults:
+* -XX:UseParallelGC (if no other garbage collector was set)
+* -ea
+* -XX:+UseCodeCacheFlushing
+* -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
This one have to be configured by your JVM or some other implicit configuration like the `JAVA_TOOL_OPTIONS` environment variable. That's not what does KGP configure by default.

Reply:
Ah, my mistake, will remove